examples/sleeper.js

```diff
@@ -6,14 +6,18 @@
  *
  * You can ask the bot to sleep or wake up by sending a chat message.
  */
-const mineflayer = require('mineflayer')
+import { createBot } from 'mineflayer'
 
 if (process.argv.length < 4 || process.argv.length > 6) {
   console.log('Usage : node sleeper.js <host> <port> [<name>] [<password>]')
   process.exit(1)
 }
 
-const bot = mineflayer.createBot({
+console.log('Commands :\n' +
+  '  cleep\n' +
+  '  wakeup')
```

Comment:
cleep?

Also: Why console log? It's not intended to be used as a bot. It's intended to act as an example of a bot. Anyone who use that file will look at the code before running it.